helm/ffc-pr-calculator/values.yaml

```diff
@@ -1,4 +1,4 @@
-environment: development
```

Comment:
This value is always overridden to `production` so no value in initially setting it to dev.  Reduces the amount of config we need by doing this.